docker/table_docker_compose_service.go

```diff
@@ -0,0 +1,512 @@
+				Name:        "cpu_percent",
+				Type:        proto.ColumnType_DOUBLE,
+				Description: "CPU utilization limit as a percentage.",
+				Transform:   transform.FromField("CPUPercent"),
+			},
+			{
+				Name:        "cpu_period",
+				Type:        proto.ColumnType_INT,
+				Description: "CPU CFS (Completely Fair Scheduler) period.",
+				Transform:   transform.FromField("CPUPeriod"),
+			},
+			{
+				Name:        "cpu_quota",
+				Type:        proto.ColumnType_INT,
+				Description: "CPU CFS (Completely Fair Scheduler) quota.",
+				Transform:   transform.FromField("CPUQuota"),
+			},
+			{
+				Name:        "cpu_rt_period",
+				Type:        proto.ColumnType_INT,
+				Description: "CPU real-time period.",
+				Transform:   transform.FromField("CPURTPeriod"),
+			},
+			{
+				Name:        "cpu_rt_runtime",
+				Type:        proto.ColumnType_INT,
+				Description: "CPU real-time runtime.",
+				Transform:   transform.FromField("CPURTRuntime"),
+			},
+			{
+				Name:        "cpus",
+				Type:        proto.ColumnType_DOUBLE,
+				Description: "Number of CPUs to allocate to the container (in fractional form).",
+				Transform:   transform.FromField("CPUS"),
+			},
+			{
+				Name:        "cpuset",
+				Type:        proto.ColumnType_STRING,
+				Description: "CPUs in which to allow execution (comma-separated list or ranges).",
+				Transform:   transform.FromField("CPUSet"),
+			},
+			{
+				Name:        "cpu_shares",
+				Type:        proto.ColumnType_INT,
+				Description: "CPU shares (relative weight) for the container.",
+				Transform:   transform.FromField("CPUShares"),
+			},
+			{
+				Name:        "command",
+				Type:        proto.ColumnType_STRING,
+				Description: "Command for the service containers.",
+			},
+			{
+				Name:        "configs",
+				Type:        proto.ColumnType_JSON,
+				Description: "List of configurations for the service.",
+			},
+			{
+				Name:        "container_name",
+				Type:        proto.ColumnType_STRING,
+				Description: "Name of the container.",
+			},
+			{
+				Name:        "credential_spec",
+				Type:        proto.ColumnType_JSON,
+				Description: "Credential specification for the container.",
+			},
+			{
+				Name:        "depends_on",
+				Type:        proto.ColumnType_JSON,
+				Description: "Dependencies for the service.",
+			},
+			{
+				Name:        "deploy",
+				Type:        proto.ColumnType_JSON,
+				Description: "Deployment configuration for the service.",
+			},
+			{
+				Name:        "device_cgroup_rules",
+				Type:        proto.ColumnType_JSON,
+				Description: "List of device cgroup rules for the container.",
+			},
+			{
+				Name:        "devices",
+				Type:        proto.ColumnType_JSON,
+				Description: "List of devices to add to the container.",
+			},
+			{
+				Name:        "dns",
+				Type:        proto.ColumnType_JSON,
+				Description: "List of DNS servers for the container.",
+				Transform:   transform.FromField("DNS"),
+			},
+			{
+				Name:        "dns_opt",
```

Comment:
Similarly, should this column be called `dns_opts`? Considering the fact that it is a JSON column wherein we can retain the plural nomenclature.